hudi-common/src/test/java/org/apache/hudi/common/model/TestPartialUpdateAvroPayload.java

```diff
@@ -155,8 +155,8 @@ public void testDeletedRecord() throws IOException {
     PartialUpdateAvroPayload payload1 = new PartialUpdateAvroPayload(record1, 0L);
     PartialUpdateAvroPayload payload2 = new PartialUpdateAvroPayload(delRecord1, 1L);
 
-    assertArrayEquals(payload1.preCombine(payload2).recordBytes, payload2.recordBytes);
-    assertArrayEquals(payload2.preCombine(payload1).recordBytes, payload2.recordBytes);
+    assertArrayEquals(payload1.preCombine(payload2, schema, new Properties()).recordBytes, payload2.recordBytes);
+    assertArrayEquals(payload2.preCombine(payload1, schema, new Properties()).recordBytes, payload2.recordBytes);
 
```

Comment:
Can you add an explicit test-case for deleted record case here during precombine. The test-case needs to check for _hoodie_is_deleted flag in the returned record.